GitUI/Translation/English.xlf

```diff
@@ -6311,35 +6311,35 @@ Would you like to do it now?</source>
         <target />
       </trans-unit>
       <trans-unit id="Abort.Text">
-        <source>Abort</source>
+        <source>A&amp;bort</source>
```

Comment:
I don’t think the level of consistency that you seem to be aspiring to here is generally attainable. The Apply Patch dialog has a “Browse” button for which B is a pretty good fit. The Rebase dialog has a “Rebase on” textfield, for which R is a pretty good fit. I don’t think keeping _all_ mnemonics perfectly consistent between all dialogs is realistic.

On this occasion, however, it is possible to swap R and B in Apply Patch. This changes &Browse to B&rowse and Abo&rt to A&bort. If you prefer this version, feel free to merge that. As a user, though, I would personally look at this and wonder “why didn’t they use B for Browse, isn’t that more obvious?”. The only reason you noticed the discrepancy is because I submitted both dialogs in the same PR, but a user wouldn’t be comparing the two dialogs like that.